### PR TITLE
Revamp actionable tabs

### PR DESCRIPTION
**Pull request by oncletom**
# Home

screen shot 2017-06-23 at 16 58 57

# Organisation

screen shot 2017-06-23 at 16 58 34

# Search

screen shot 2017-06-24 at 00 22 03

Contains code which can be levered by opendatateam/udata#999

searchbar gif

### CODE REVIEW

**Comment by davidbgk** [comment]
More space between tabs? Centered tabs? (just to visualize it)

**Comment by oncletom** [comment]
# Centered Tabs

screen shot 2017-06-23 at 17 19 39
screen shot 2017-06-23 at 17 20 22

**Comment by oncletom** [comment]
# More Margins

screen shot 2017-06-23 at 17 23 21

**Comment by davidbgk** [comment]
👍 for margins, not sure for centering.

I think we can now [check all boxes here](https://www.nngroup.com/articles/tabs-used-right/) 😉

**Comment by thimy** [comment]
Left-aligned tabs seems better to me :)

**Comment by oncletom** [comment]
I agree with the left align although centered is not so bad either.

I amended the PR to make it work nicely with search tabs. opendatateam/udata#999 makes it nicer with the ability to target specifically the breadcrumb area of the search results page.

File: udata_gouvfr/theme/templates/subnav-small.html [hunk 1]
```diff
@@ -22,19 +22,6 @@
...
-                </button>
-                <ul class="dropdown-menu" role="menu" aria-labelledby="topics">
-                    {% for topic in g.featured_topics %}
-                    <li role="presentation">
-                        <a role="menuitem" tabindex="-1" href="{{ url_for('topics.display', topic=topic) }}">
-                            {{ topic.name }}
-                        </a>
-                    </li>
-                    {% endfor %}
-                </ul>
```
Comment (noirbizarre) [inline udata_gouvfr/theme/templates/subnav-small.html]: I think this removal is not related to this PR (and should be here)

Comment (oncletom) [inline udata_gouvfr/theme/templates/subnav-small.html]: Good catch. It was clearly not meant to be part of this PR 🙂

**Comment by oncletom** [comment]
@noirbizarre I have amended the commits to remove the unwanted change.

I have kept the _left alignment_ everywhere for now — I'd rather focus on just making the tabs more visible and consider the items alignment in a more _featuresque_ PR with the help of @thimy to _design_ improvements with a bit more of background research if you are okay with it.

Here are the screenshots when in mobile mode. Before and after are similar but I can feel it is not optimal. I would definitely reconsider item ordering to make the form actionable elements — it is connected to the previous point.

# Before

screen shot 2017-06-26 at 14 35 40


# After

screen shot 2017-06-26 at 14 35 33

**Comment by davidbgk** [comment]
About search results, it's not clear to me these are tabs (vs. buttons for instance) or which one is selected. What about having the same pattern as other (new) tabs with borders?

**Comment by oncletom** [comment]
@davidbgk I can pimp a different border colour to check how it goes to both reflect the content tabs and to merge smoothly within the breadcrumb colour code 🙂
